interp/notation_ops.ml

```diff
@@ -1003,15 +1003,15 @@ let rec match_ inner u alp metas sigma a1 a2 =
 
   (* Matching individual binders as part of a recursive pattern *)
   | GLambda (_,Name p,bk,t,GCases (_,LetPatternStyle,None,[(GVar(_,e),_)],[(_,_,[cp],b1)])),
-    NLambda (Name id,_,b2)
+    NLambda (Name id,_,_,b2)
       when is_bindinglist_meta id metas ->
       let alp,sigma = bind_bindinglist_env alp sigma id [(Inr cp,bk,None,t)] in
       match_in u alp metas sigma b1 b2
-  | GLambda (_,na,bk,t,b1), NLambda (Name id,_,b2)
+  | GLambda (_,na,bk,t,b1), NLambda (Name id,_,_,b2)
       when is_bindinglist_meta id metas ->
       let alp,sigma = bind_bindinglist_env alp sigma id [(Inl na,bk,None,t)] in
       match_in u alp metas sigma b1 b2
-  | GProd (_,na,bk,t,b1), NProd (Name id,_,b2)
+  | GProd (_,na,bk,t,b1), NProd (Name id,_,_,b2)
       when is_bindinglist_meta id metas && na != Anonymous ->
```

Comment:
Idem for these case where the `NLambda` and `NProd` are placeholders for a sequence of binders.